dm_control/locomotion/walkers/assets/dog_v2/add_torque_actuators.py

```diff
@@ -58,11 +58,19 @@ def add_motors(physics, model, lumbar_joints, cervical_joints, caudal_joints):
   all_spinal_joints = []
   for region in spinal_joints.values():
     all_spinal_joints.extend(region)
-  root_joint = model.find('joint', 'root')
+
+  # root_joint = model.find('joint', 'root')
```

Comment:
I'm not happy about the free joint -> euler angle decision. Why did you need this again? This will definitely become a problem for large rotations (e.g. rolling on the floor).

I think we should understand whatever the underlying cause was and fix it, rather than do this.